maintenance/projects/js-toolkit/packages/liferay-npm-build-support/src/config.js

```diff
@@ -99,6 +103,22 @@ export function getSupportedLocales() {
 	return npmbuildrc.supportedLocales;
 }
 
+/**
+ * Get the `npm run start` webpack's directory
+ * @return {string}
+ */
+export function getStartDir() {
+	return npmbuildrc.start.dir;
+}
+
+/**
+ * Check if `npm run start` configuration was ejected
+ * @return {boolean}
+ */
+export function isStartEjected() {
+	return npmbuildrc.start.ejected;
```

Comment:
Same as above

> Do we need to add any optional chaining here or can we guarantee that these keys exist on the object?

Reply:
We can, because of this: https://github.com/liferay/liferay-frontend-projects/pull/526/files#diff-6ef61c9a84a94dbc3c6aa6df2a6bc3ebfe6544090c6a686132b62755775a9211R24

Reply:
In general, I try to normalize all configuration to its canonical form, plus default values. This is done for `.npmbundlerrc` too. For example, when there are some equivalent settings, I designate one of them as the canonical form and transform the others right after reading the `.npmbundlerrc` file so that the application code doesn't have to deal with different places.

Reply:
> In general, I try to normalize all configuration to its canonical form, plus default values

👍 👍  I figured you were normalizing the config somewhere, but wanted to just make sure. Thanks for the clarification